bigchaindb/util.py

```diff
@@ -199,7 +199,9 @@ def verify_signature(signed_transaction):
     if 'assignee' in data:
         data.pop('assignee')
 
-    signature = data.pop('signature')
+    signature = data.pop('signature', None)
```

Comment:
I've made this suggestion before.
Recalled that @vrde might wanted to raise a KeyError